### DISCUSSION TITLE
java.lang.NoSuchFieldError: INSTANCE

### DISCUSSION THREAD

**Discussion by chanvoyage** [strimzi/strimzi-kafka-operator#5642] [Q&A]
I'm trying to get the SpoolDirCsvSourceConnector  follow up https://rmoff.net/2020/06/17/loading-csv-data-into-kafka/ 
,but  found something error like this 

<img width="1397" alt="Screen Shot 2564-10-03 at 15 09 26" src="https://user-images.githubusercontent.com/39899497/135745661-67b7dfde-50a8-40ba-923a-c64a4b4e2f88.png">

======csv connect yaml=========

apiVersion: kafka.strimzi.io/v1beta2
kind: KafkaConnector
metadata:
  name: csv-connector
  namespace: kafka
  labels:
    strimzi.io/cluster:  thdlcd0-dev-connect-cluster
spec:
  class: "com.github.jcustenborder.kafka.connect.spooldir.SpoolDirCsvSourceConnector"
  tasksMax: 1
  config:
    topic: csv-testing-topic
    input.path: /home/kafka/data/unprocessed
    error.path: /home/kafka/data/error
    finished.path: /home/kafka/data/precessed
    input.file.pattern: ".*\\.csv"
    schema.generation.enabled: true
    csv.first.row.as.header: true
================== Check my plugin  ===========

![image](https://user-images.githubusercontent.com/39899497/135745748-18d0061e-ecdb-46fe-9a14-df963207e2c3.png)
========= path csv file ========
![image](https://user-images.githubusercontent.com/39899497/135745781-90680b45-c6dc-4ced-9ab0-5445bb75fb06.png)

====== path plugin connector========

![image](https://user-images.githubusercontent.com/39899497/135745916-cd5145ac-7dad-4cbd-b7c1-b0e3a606d487.png)

**Comment by scholzj** [comment]
The Exception seems to be coming from, the connector, so you probably need to start the investigation there.
